dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -4117,6 +4117,25 @@ paths:
               $ref: "#/definitions/Organization"
       security:
       - BEARER: []
+  /users/starredServices:
```

Comment:
going to have to work with Elizabeth on this one. but if the swagger.yaml isn't needed, then you can omit this with `@ApiOperation(value = "hidde", hidden=true` or something like that

Reply:
Spoke with Elizabeth and she needs it so I kept the swagger.yaml